cyral/resource_cyral_policy_rule_test.go

```diff
@@ -63,6 +63,7 @@ func TestAccPolicyRuleResource(t *testing.T) {
 				Config: testUpdate2Config,
 				Check:  testUpdate2Func,
 			},
+			// TODO: add import test -aholmquist 2022-08-01
```

Comment:
Same comment above, we can use the `ImportStateId`, and this can be done in a separate PR in the future.